packages/contractkit/src/web3-contract-cache.ts

```diff
@@ -0,0 +1,102 @@
+import { CeloContract } from 'src/base'
+import { newAttestations } from 'src/generated/Attestations'
+import { newBondedDeposits } from 'src/generated/BondedDeposits'
+import { newEscrow } from 'src/generated/Escrow'
+import { newExchange } from 'src/generated/Exchange'
+import { newGasCurrencyWhitelist } from 'src/generated/GasCurrencyWhitelist'
+import { newGasPriceMinimum } from 'src/generated/GasPriceMinimum'
+import { newGoldToken } from 'src/generated/GoldToken'
+import { newGovernance } from 'src/generated/Governance'
+import { newMultiSig } from 'src/generated/MultiSig'
+import { newRandom } from 'src/generated/Random'
+import { newRegistry } from 'src/generated/Registry'
+import { newReserve } from 'src/generated/Reserve'
+import { newSortedOracles } from 'src/generated/SortedOracles'
+import { newStableToken } from 'src/generated/StableToken'
+import { newValidators } from 'src/generated/Validators'
+import { ContractKit } from 'src/kit'
+
+const ContractFactories = {
```

Comment:
what's the difference btwn this file and `contract-cache`?

Reply:
We have 2 types of contract objects: web3 wrappers, and our own wrappers.

As discussed in the sdk meeting, we want to favor ours, and leave others accessible but not on the front seat.